mods/ra/maps/soviet-11a/soviet11a-AI.lua

```diff
@@ -0,0 +1,242 @@
+--[[
+   Copyright 2007-2021 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+FirstPatrolActors = { Cruiser1, Gunboat1, Gunboat2 }
+PatrolA = { Rocket1, Rocket2 }
+PatrolB = { Rocket3, Rocket4 }
+PatrolC = { Rocket5, Rocket6 }
+PatrolPathA = { RocketPath1.Location, RocketPath2.Location }
+PatrolPathB = { RocketPath3.Location, RocketPath4.Location }
+PatrolPathC = { RocketPath5.Location, RocketPath6.Location }
+
+Helipads = { Helipad1, Helipad2, Helipad3, Helipad4 }
+HeliType = { "heli" }
+DestroyerSquad = { "dd", "dd" }
+
+DestroyerDelays =
+{
+	easy = 4,
+	normal = 3,
+	hard = 2
+}
+
+TransportDelays =
+{
+	easy = 6,
+	normal = 5,
+	hard = 4
+}
+
+AlliedShips =
+{ 
+	easy = { "pt", "pt", "dd" },
+	normal = { "pt", "dd", "dd" },
+	hard = { "dd", "dd" , "dd"}
```

Comment:
Style nit: there's a spurious space after the 2nd and missing space after the 3rd entry now.